packages/firestore/test/integration/util/helpers.ts

```diff
@@ -213,11 +215,11 @@ export function withTestDbsSettings(
       return wipeDb(dbs[0]).then(() =>
         dbs.reduce(
           (chain, db) =>
-            chain.then(
-              db.INTERNAL.delete.bind(this, {
-                purgePersistenceWithDataLoss: true
-              })
-            ),
+            chain
```

Comment:
It's worth rewriting gunk like this as an async function rather than piling more crazy thens in here.